kernel/kernel.c

```diff
@@ -16,52 +16,59 @@
 #include "memory.c"
 #include "interrupts.c"
 #include "shell.c"
+#include "util/printer.c"
```

Comment:
Did you forget to commit this file, I don't see it anywhere

Reply:
Good point